QA_Robot_Practice/pages/res/LoginPageLocators.robot

```diff
@@ -0,0 +1,5 @@
+*** Variables ***
+${LOGIN_INPUT_FIELD_LOCATOR}    j_username
+${PASSWORD_INPUT_FIELD_LOCATOR}    j_password
+${SUBMIT_BUTTON_LOCATOR}    //button[@type='submit']
+${ERROR_CASE_ELEMENT}     //font[@color='red']
```

Comment:
It means any red element will be considered as an error message. Locator should identify needed element. In other case we will have a false-positive test